classes/patterns/class-action.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Action pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Action.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Action extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/action';
```

Comment:
`-template` makes sense but also [means](https://developer.wordpress.org/themes/basics/template-files/) [something](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-templates/) in wordpress :sob: I'll add my proposition to the confusion `-page-content`

Reply:
`-page-pattern` sounds good to me! @lithrel what do you think?
Whichever we decide, we will also need to update the [Get Informed template](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/classes/patterns/class-getinformed.php#L22) that has already been merged 😬

Reply:
Good for me :+1:

Reply:
Thank you both for your input! [Updated](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/879/commits/01a63331e270a4ab26a1b2632fbce3bac96640f8) 😄

Reply:
Hey! Coming late to the <s>party</s> discussion. There is actually a proposal by suzi to name these as "Layout Block Patterns" (or "Layouts" for short), because "Templates" has indeed a different meaning in WP and "Pages" is just one post type.

So maybe we could use `-layout-pattern` instead :grin:

Reply:
I'm fine with either of those 🤷‍♀️

Reply:
[Updated](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/879/commits/7040332e5b4649d2ead50cc1e66d11a1f5439598) after confirming with @GP-Dan-Tovbein